Description: Set X-LightDM-DesktopName to GNOME for GNOME session.
Bug: https://bugzilla.gnome.org/727546
Bug-Ubuntu: https://launchpad.net/bugs/1212408
Author: Tim Lunn <tim@feathertop.org>

diff -pruN -x '*~' gnome-shell-extensions-3.14.1.orig/data/gnome-classic.desktop.in gnome-shell-extensions-3.14.1/data/gnome-classic.desktop.in
--- gnome-shell-extensions-3.14.1.orig/data/gnome-classic.desktop.in	2014-10-07 13:54:01.000000000 +0200
+++ gnome-shell-extensions-3.14.1/data/gnome-classic.desktop.in	2014-10-28 23:06:38.915410738 +0100
@@ -5,4 +5,5 @@ Exec=env GNOME_SHELL_SESSION_MODE=classi
 TryExec=gnome-session
 Icon=
 Type=Application
+X-LightDM-DesktopName=GNOME
 DesktopNames=GNOME-Classic;GNOME
